doc/util.md

```diff
@@ -25,6 +25,18 @@ This method sets the orientation of the whole application (effectively, also the
 
 If a finer control of the allowed orientations is required (without having to deal with `SystemChrome` directly), `setOrientation` (accepts a single `DeviceOrientation` as a parameter) and `setOrientations` (accepts a `List<DeviceOrientation>` for possible orientations) can be used.
 
+### `Flame.util.decodeImageFromPixels()`
+
+The [dart-ui decodeImageFromPixels](https://api.flutter.dev/flutter/dart-ui/decodeImageFromPixels.html) currently does not support the web platform. So if you are looking for a way to manipulate pixel data on the web this method can be used as a replacement for `dart-ui decodeImageFromPixels`:
+
+```dart
+final image = await Flame.util.decodeImageFromPixels(
```

Comment:
Just for making it easy to  understand, I would put the type here, so it is clear that this method returns an Image.

Reply:
Agreed.